core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -248,7 +251,7 @@ public static String getJournalDirectory(String baseDirectory) {
    */
   public FileSystemMaster(BlockMaster blockMaster, Journal journal) {
     this(blockMaster, journal, ExecutorServiceFactories
-        .fixedThreadPoolExecutorServiceFactory(Constants.FILE_SYSTEM_MASTER_NAME, 2));
+        .fixedThreadPoolExecutorServiceFactory(Constants.FILE_SYSTEM_MASTER_NAME, 3));
```

Comment:
Yes, the thread will just be left in the pool. An alternative way would be to use a separate thread, but I think this thread can be reused in the future if we want to do something like update the status.